wordpress-mobile/WordPress-Android
# [Revamp Landing Screen] Jetpack App - Set fixed font size for animated text

Fixes #17299

To test:
<details>
<summary><b>Prerequisite</b> - Enable the <code>LandingScreenRevamp</code> feature flag</summary>

- Either `login` → `Me` → `App Settings` → `Debug Settings` → enable `LandingScreenRevamp` → `RESTART THE APP` → `logout` or
- Apply this hard-coded solution:

<details>
<summary>Patch to enable the <code>LANDING_SCREEN_REVAMP</code> build flag</summary>

```diff
diff --git a/WordPress/build.gradle b/WordPress/build.gradle
index 63260a4ca8..fe999fa92f 100644
--- a/WordPress/build.gradle
+++ b/WordPress/build.gradle
@@ -112,7 +112,7 @@ android {
         buildConfigField "boolean", "READER_COMMENTS_MODERATION", "false"
         buildConfigField "boolean", "SITE_INTENT_QUESTION", "true"
         buildConfigField "boolean", "SITE_NAME", "false"
-        buildConfigField "boolean", "LANDING_SCREEN_REVAMP", "false"
+        buildConfigField "boolean", "LANDING_SCREEN_REVAMP", "true"
         buildConfigField "boolean", "LAND_ON_THE_EDITOR", "false"
         buildConfigField "boolean", "BLOGGING_PROMPTS", "false"
         buildConfigField "boolean", "QUICK_START_EXISTING_USERS_V2", "false"
```  
</details>

---

</details>

1. Launch Jetpack App and make sure you're logged out
2. Set font size to max large / small via Accessibility settings on your device
3. **Expect** The font size of the animated text to **not** change

## Previews

| Smallest Font Size | Largest Font Size |
| - | - |
| <img width="314" src="https://user-images.githubusercontent.com/4588074/195139494-1f57e594-1fd9-49a4-8680-3d634c6a3576.png"> | <img width="314" src="https://user-images.githubusercontent.com/4588074/195139510-0af1e992-a3ec-4395-af39-9033b3deb3d0.png"> |

## Regression Notes
1. Potential unintended areas of impact
   N/a

3. What I did to test those areas of impact (or what existing automated tests I relied on)
   N/a

4. What automated tests I added (or what prevented me from doing so)
   N/a

PR submission checklist:
